metricbeat/module/system/process/_meta/docs.asciidoc

```diff
@@ -17,3 +17,21 @@ On Linux this metricset will collect metrics from any cgroups that the process
 is a member of. This feature is enabled by default and can be disabled by adding
 `process.cgroup.enabled: false` to the system module configuration.
 
+[float]
+=== Process Environment Variables
+
+This metricset can collect the environment variables that were used to start the
+process. This feature is available on Linux, Darwin, and FreeBSD. By default
+no environment variables are collected. You must configure the environment
```

Comment:
We could mention here, that for security reasons no env variables are fetched by default.